src/vt.tsx

```diff
@@ -529,6 +527,8 @@ const VTable: React.ForwardRefRenderFunction<RefObject, VTableProps> = (props, r
           target: {
             scrollTop: top,
             scrollLeft: target.scrollLeft,
+            scrollHeight: target.scrollHeight,
+            clientHeight: target.clientHeight,
           },
           end: target.scrollHeight - target.clientHeight === Math.round(top),
```

Comment:
I strongly recommend you to keeping it simple rather than including another approach.
so we can do like this
```js
Math.abs(target.scrollHeight - target.clientHeight - Math.round(top)) < 3
```